Python/Twitter_Scrapper/README.md

```diff
@@ -0,0 +1,26 @@
+## Twitter Scrapper
+
+This script extracts all Tweets, Retweets, Images, Videos, Hashtags, Likes and Pinned tweet of a specific user in a .csv file.
+
+![How to use the script](https://i.ibb.co/mcjK65f/Tweeter-Scrapper.png)
```

Comment:
Can you please add the snapshot on the bottom side of the documentation ?

Reply:
Add the image on the bottom side